components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/services/SessionManagementService.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.application.authentication.framework.services;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+import org.wso2.carbon.core.AbstractAdmin;
+import org.wso2.carbon.identity.application.authentication.framework.cache.SessionContextCache;
+
+public class SessionManagementService extends AbstractAdmin {
+
+    private static Log log = LogFactory.getLog(SessionManagementService.class);
+
+    public String removeSession(String sessionId) {
+
+        if (StringUtils.isBlank(sessionId)) {
+            return "Invalid Session ID";
```

Comment:
This returning String is ugly. Better to go with a Boolean type or something which provides easy capability to decide whether the operation was success or not for the calling party.